package.json

```diff
@@ -12,16 +12,29 @@
     "Typescript",
     "Microsoft"
   ],
-  "main": "index.js",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "tsc"
+  },
+  "files": [
+    "README.md",
+    "dist/index.js",
+    "dist/index.d.ts"
+  ],
   "repository": {
     "type": "git",
     "url": "https://github.com/Microsoft/Typedoc-Webpack-Plugin.git"
   },
   "dependencies": {
     "lodash.clone": "^4.5.0",
-    "lodash.merge": "^4.6.0"
+    "lodash.merge": "^4.6.0",
+    "typedoc": "^0.11.1"
```

Comment:
I bundled typedoc as a dependency though it was originally just a peer dependency.
Which do you think is a better approach?